wandb/proto/wandb_internal.proto

```diff
@@ -384,6 +384,7 @@ message ArtifactRecord {
   string sequence_client_id = 16;
   string base_id = 17;
   int64 ttl_duration_seconds = 18;
+  string staging_dir = 19;
```

Comment:
one thing to note that records will be persistent in the transaction log.
so make sure to handle any breaking changes, i.e. trying to sync older `wandb` file with newer version of nexus.